### PR TITLE
Add main content structure

### PR DESCRIPTION
**Pull request by maya**
This adds basic main content structure and styling for the library.

This resolves the third item in: https://github.com/18F/govt-wide-patternlibrary/issues/21

Also adds the basic structure of the sidebar, although it is currently empty.

This is what it looks like:

screen shot 2015-06-17 at 3 56 26 pm

### CODE REVIEW

**Comment by juliaelman** [comment]
@maya I just pushed up a fix. Please let me know if you have any other issues or have any questions!

**Comment by maya** [comment]
@juliaelman amazing, thanks so much!

**Comment by maya** [comment]
The last commits add structure and placeholder text for "Use" and "Accessibility" for all collections. It also adds basic structure for our preview examples. This removes sample content that was in a few collections and used for testing.

This is what it looks like:

main-content

**Comment by juliaelman** [comment]
Great! Is this PR ready for review?

**Comment by maya** [comment]
@juliaelman yep! I just removed the WIP tags from the PR.

File: _includes/navbar.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<div class="nav-sidebar">
+</div>
```
Comment (juliaelman) [inline _includes/navbar.html]: I think this should be located in `_includes/sidenav.html`, as the purpose of this include is for the main top navigation.

**Comment by juliaelman** [comment]
Looks good! One small change, but otherwise feel free to :rocket:

**Comment by maya** [comment]
You're right! Just fixed that in 3749066.

**Pull request merged**